sentry/src/main/java/io/sentry/SentryEnvelopeItem.java

```diff
@@ -243,6 +204,87 @@ public static SentryEnvelopeItem fromAttachment(
     return new SentryEnvelopeItem(itemHeader, () -> cachedItem.getBytes());
   }
 
+  public static @Nullable SentryEnvelopeItem fromProfilingTrace(
+      final @NotNull ProfilingTraceData profilingTraceData,
+      final long maxTraceFileSize,
+      final @NotNull ISerializer serializer)
+      throws SentryEnvelopeException {
+
+    File traceFile = profilingTraceData.getTraceFile();
+    if (!traceFile.exists()) {
+      return null;
+    }
+
+    // The payload of the profiling item is a json that includes the trace file encoded with base64
```

Comment:
Usually, base64 is 1/3 bigger, is there a reason to do so? Envelopes are able to send binary format.

Reply:
We agreed with @phacops, our backend guy, that will expect a base64 string.
Also, the payload of the profile envelope is a json, so a string is a natural fit

Reply:
I see, since it's a file, I was expecting to be an attachment that could be sent in bytes.
Reading the code now I understand, the envelope type is `profile`, which is a JSON of the `ProfilingTraceData` class, that contains a `stacktrace` field, and this field has the profiling file content in base64.

Reply:
yep